extra/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@
 configure_file(nrnivmodl_core_makefile.in
                ${CMAKE_BINARY_DIR}/share/coreneuron/nrnivmodl_core_makefile @ONLY)
 configure_file(nrnivmodl-core.in ${CMAKE_BINARY_DIR}/bin/nrnivmodl-core @ONLY)
+add_custom_target(
+  nrnivmodl-core ALL
+  DEPENDS ${CMAKE_BINARY_DIR}/bin/nrnivmodl-core
```

Comment:
A minor comment is that I don't **think** this is enough for changes to `nrnivmodl-core.in` to trigger build rules that run `nrnivmodl-core` to rerun. It might be nice to acknowledge that in a comment.